include/particles/particle.tcc

```diff
@@ -237,7 +237,9 @@ void mpm::Particle<Tdim>::initialise() {
 
   // Initialize vector data properties
   this->properties_["stresses"] = [&]() { return stress(); };
+  this->properties_["shear_stresses"] = [&]() { return shear_stress(); };
```

Comment:
Could you please remind me where this is called?

Reply:
First in `mpm-base.tcc` [here](https://github.com/cb-geo/mpm/blob/0675471fceb83d95afae78ca5cc4ffb7f7bcf2ad/include/solvers/mpm_base.tcc#L502), then in `mesh.tcc` [here](https://github.com/cb-geo/mpm/blob/0675471fceb83d95afae78ca5cc4ffb7f7bcf2ad/include/mesh.tcc#L934)